test/factories/advertisements.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :advertisement do
+    title { "title" + ('a'..'z').to_a.sample(26).join }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.